packages/sdk-utils/src/percy-enabled.js

```diff
@@ -4,16 +4,17 @@ import request from './request';
 
 // Create a socket to connect to a remote logger
 async function connectRemoteLogger() {
-  await logger.remote(() => {
+  await logger.remote(async () => {
     let url = percy.address.replace('http', 'ws');
 
     if (process.env.__PERCY_BROWSERIFIED__) {
       return new window.WebSocket(url);
     } else {
-      let socket = new (require('ws'))(url);
+      /* eslint-disable-next-line import/no-extraneous-dependencies */
+      let { default: WebSocket } = await import('ws');
```

Comment:
While here, I also switched this to a dynamic import rather than a commonjs require in preparation for a full ES module adaptation in the future (dynamic imports will work in the current commonjs environment)